snap/snapcraft.yaml

```diff
@@ -25,37 +25,22 @@ apps:
       - dbus-daemon
     common-id: com.github.unrud.VideoDownloader
     desktop: usr/share/applications/com.github.unrud.VideoDownloader.desktop
+    environment:
+      PATH: $SNAP_DESKTOP_RUNTIME/usr/bin:$PATH
```

Comment:
This has no effect. `/snap/video-downloader/current/usr/bin/python3` is still used.